Description: reset the variables in printf_parser to NULL.

the PrintfParser variables g1Xg2Xg3 and wg_offset should be reinit
after the builder is deleted, or else the variables will be freed and
caused memory leak or crash

Author: Luo Xionghu <xionghu.luo@intel.com>
Origin: upstream http://cgit.freedesktop.org/beignet/commit/?id=dcc189bed5b395d6fbcdffa29e60e68cf1071310
Bug: https://bugs.freedesktop.org/show_bug.cgi?id=90472

diff --git a/backend/src/llvm/llvm_printf_parser.cpp b/backend/src/llvm/llvm_printf_parser.cpp
index 1e8427c..bdaed8a 100644
--- a/backend/src/llvm/llvm_printf_parser.cpp
+++ b/backend/src/llvm/llvm_printf_parser.cpp
@@ -723,6 +723,8 @@ error:
 
     deadprintfs.clear();
     delete builder;
+    g1Xg2Xg3 = NULL;
+    wg_offset = NULL;
 
     return changed;
   }

